src/matchers/toSatisfyAll.js

```diff
@@ -1,23 +1,28 @@
 export function toSatisfyAll(actual, expected) {
   const { printReceived, printExpected, matcherHint } = this.utils;
 
+  const pass = actual.every(expected);
+  const failingElement = pass ? actual.find(expected) : actual.find(a => !expected(a));
```

Comment:
There could be multiple failing elements, right? I'm wondering if it'd be better to print them all instead of just the first one.